infra/aws/terraform/prow-build-cluster/locals.tf

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+locals {
+  root_account_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:root"
+
+  # TODO(xmudrii): This is a temporary condition. To be deleted after making canary cluster a build cluster.
```

Comment:
Let's remove this TODO, we don't have plans to make this cluster a Prow build cluster.